pkg/resources/management.cattle.io/v3/globalrole/validator.go

```diff
@@ -264,3 +269,35 @@ func (a *admitter) validateUpdateFields(oldRole, newRole *v3.GlobalRole, fldPath
 	}
 	return nil
 }
+
+func (a *admitter) validateNamespacedRules(request *admission.Request, rules map[string][]v1.PolicyRule) []error {
+	var returnErrs []error
+	// check for escalation in each namespace of the GlobalRole's NamespacedRules
+	for namespace, rules := range rules {
+		// check that namespace exists
+		_, err := a.namespaces.Get(namespace)
+		if err != nil {
+			returnErrs = append(returnErrs, fmt.Errorf("error getting namespace %v: %w", namespace, err))
+			continue
+		}
```

Comment:
I thought about this a bit more and realized that this is poor UX. I think that we can use the status field/the controllers to indicate when a user has attempted to grant permissions in a non-existent namespace. That combined with the enqueuer on the namespace should do fine. Would you mind removing the check on Namespace existence for now?